BE/src/main/java/kr/codesquad/issuetracker/web/dto/milestone/MilestoneDto.java

```diff
@@ -0,0 +1,45 @@
+package kr.codesquad.issuetracker.web.dto.milestone;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import kr.codesquad.issuetracker.domain.milestone.Milestone;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.springframework.format.annotation.DateTimeFormat;
+
+@Getter
+@AllArgsConstructor
+public class MilestoneDto {
+
+	private Long id;
+
+	private String title;
+
+	private String description;
+
+	@DateTimeFormat(pattern = "yyyy-MM-dd hh:mm:ss")
+	private LocalDateTime createdDateTime;
+
+	@DateTimeFormat(pattern = "yyyy-MM-dd hh:mm:ss")
+	private LocalDateTime modifiedDateTime;
+
+	@DateTimeFormat(pattern = "yyyy-MM-dd hh:mm:ss")
+	private LocalDate deadLine;
+
+	private int openedIssue;
+
+	private int closedIssue;
+
+	public static MilestoneDto of(Milestone milestone) {
```

Comment:
[이펙티브 자바 아이템 1](https://sun-22.tistory.com/84)

```suggestion
	public static MilestoneDto from(Milestone milestone) {
```

Reply:
생각없이 of만 사용을 했는데 이런 부분이 있었군요 감사합니다 :)